services/proxy/pkg/metrics/metrics.go

```diff
@@ -14,33 +14,33 @@ var (
 
 // Metrics defines the available metrics of this service.
 type Metrics struct {
-	Counter   *prometheus.CounterVec
-	Latency   *prometheus.SummaryVec
+	Requests  *prometheus.CounterVec
+	Errors    *prometheus.CounterVec
 	Duration  *prometheus.HistogramVec
 	BuildInfo *prometheus.GaugeVec
 }
 
 // New initializes the available metrics.
 func New() *Metrics {
 	m := &Metrics{
-		Counter: prometheus.NewCounterVec(prometheus.CounterOpts{
+		Requests: prometheus.NewCounterVec(prometheus.CounterOpts{
 			Namespace: Namespace,
 			Subsystem: Subsystem,
-			Name:      "proxy_total",
-			Help:      "How many proxy requests processed",
-		}, []string{}),
-		Latency: prometheus.NewSummaryVec(prometheus.SummaryOpts{
+			Name:      "requests_total",
+			Help:      "How many requests processed in total",
+		}, []string{"method"}),
+		Errors: prometheus.NewCounterVec(prometheus.CounterOpts{
 			Namespace: Namespace,
 			Subsystem: Subsystem,
-			Name:      "proxy_latency_microseconds",
-			Help:      "proxy request latencies in microseconds",
-		}, []string{}),
+			Name:      "errors_total",
+			Help:      "How many requests run into errors",
+		}, []string{"status"}),
 		Duration: prometheus.NewHistogramVec(prometheus.HistogramOpts{
 			Namespace: Namespace,
 			Subsystem: Subsystem,
-			Name:      "proxy_duration_seconds",
-			Help:      "proxy method request time in seconds",
-		}, []string{}),
+			Name:      "duration_seconds",
+			Help:      "request duration in seconds",
+		}, []string{"method"}),
 		BuildInfo: prometheus.NewGaugeVec(prometheus.GaugeOpts{
 			Namespace: Namespace,
 			Subsystem: Subsystem,
```

Comment:
Why do you want to rename all the graphs? Taking a look at https://prometheus.io/docs/practices/naming/

> ## Metric names
> [...]
> * ...should have a suffix describing the unit, in plural form. Note that an accumulating count has total as a suffix, in addition to the unit if applicable.
>
>     * http_request_duration_**seconds**
>     * node_memory_usage_**bytes**
>     * http_requests_**total** (for a unit-less accumulating count)
>     * process_cpu_**seconds_total** (for an accumulating count with unit)
>     * foobar_build_**info** (for a pseudo-metric that provides [metadata](https://www.robustperception.io/exposing-the-software-version-to-prometheus) about the running binary)
>     * data_pipeline_last_record_processed_**timestamp_seconds** (for a timestamp that tracks the time of the latest record processed in a data processing pipeline)
> [...]
>
> ## Labels
> 
> Use labels to differentiate the characteristics of the thing that is being measured:
> 
>    * api_http_requests_total - differentiate request types: operation="create|update|delete"
>    * api_request_duration_seconds - differentiate request stages: stage="extract|transform|load"
> 
> Do not put the label names in the metric name, as this introduces redundancy and will cause confusion if the respective labels are aggregated away.

I think that our service prefixes, eg. `proxy_` should stay in place. Then again ... we are not doing it for `build_info` below.